<strong>Exception handling in Python<strong>